src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -56,6 +56,7 @@ api_platform:
         Sylius\Bundle\ApiBundle\Exception\ShippingMethodCannotBeRemoved: 422
         Sylius\Bundle\ApiBundle\Exception\ZoneCannotBeRemoved: 422
         Sylius\Bundle\ApiBundle\Exception\CannotRemoveMenuTaxonException: 409
+        Sylius\Bundle\LocaleBundle\Checker\Exception\LocaleIsUsedException: 422
```

Comment:
Wdyt?
```suggestion
        Sylius\Bundle\LocaleBundle\Checker\Exception\LocaleInUseException: 422
```

Reply:
As we don't have any standard for this, I don't have any strong opinion. I guess both names are correct.

I left the initial name, but I'm leaving this suggestion unresolved for any other opinions.